be/src/formats/parquet/column_converter.cpp

```diff
@@ -382,8 +382,8 @@ Status ColumnConverterFactory::create_converter(const ParquetField& field, const
         break;
     }
     case tparquet::Type::type::INT96: {
+        need_convert = true;
```

Comment:
Is it necessary to perform the convert for INT96 for any types? The convert function will cause some cpu overhead from my tests a few days ago, especially with cache. Perhaps set `need_convert` flag only for string type and datetime is more proper.

Reply:
if you look close to the pattern
- set `need_converter` for incompatible type
- set `converter` only when type conversion has been done by us
- and check `need_convert` and `converter` at final phase

It's just the old pattern. It's not very clear but clear enough.

So int96 is only compatible to datetime.

```
        if (col_type == PrimitiveType::TYPE_DATETIME) {
            auto _converter = std::make_unique<Int96ToDateTimeConverter>();
            RETURN_IF_ERROR(_converter->init(timezone));
            *converter = std::move(_converter);
        }
        break;

    if (need_convert && *converter == nullptr) {
        return Status::NotSupported(
                strings::Substitute("parquet column reader: not supported convert from parquet `$0` to `$1`",
                                    ::tparquet::to_string(parquet_type), type_to_string(col_type)));
    }
```